public/js/module.js

```diff
@@ -53,10 +28,58 @@ function openBubble(uniqueMarker) {
     google.maps.event.trigger(uniqueMarker, 'click');
 }
 
+function getModuleMarker(module_id) {
+  $.ajax({
+      url: "/module",
+      type: "POST",
+      data: encodeURIComponent("module_id")+"="+encodeURIComponent(module_id),
```

Comment:
The same route is being used by the module to get the marker belonging to the module. I had some cross-domain-trouble which was rectified by sending url-encoded data instead of json.